src/XMakeBuildEngine/Evaluation/Preprocessor.cs

```diff
@@ -166,13 +168,16 @@ private void CloneChildrenResolvingImports(XmlNode source, XmlNode destination)
                 {
                     // To display what the <Import> tag looked like
                     string importCondition = ((XmlElement)child).GetAttribute(XMakeAttributes.condition);
+                    string condition = importCondition.Length > 0 ? $" Condition=\"{importCondition}\"" : String.Empty;
                     string importProject = ((XmlElement)child).GetAttribute(XMakeAttributes.project);
+                    string importSdk = ((XmlElement)child).GetAttribute(XMakeAttributes.sdk);
 
                     IList<ProjectRootElement> resolvedList;
                     if (!_importTable.TryGetValue(child as XmlElement, out resolvedList))
                     {
                         // Import didn't resolve to anything; just display as a comment and move on
-                        string closedImportTag = "<Import Project=\"" + importProject + "\"" + ((importCondition.Length > 0) ? " Condition=\"" + importCondition + "\"" : String.Empty) + " />";
+                        string closedImportTag =
+                            $"<Import Project=\"{importProject}\"{condition} />";
```

Comment:
Should this also cover the case where there is an `Sdk` attribute on the import?